example/lib/main.dart

```diff
@@ -1,405 +1,69 @@
+import 'dart:io';
+
```

Comment:
I cannot run this on Chrome, probably because web apps cannot use `dart:io`.

Maybe use a string literal like earlier?

Reply:
Done